jupyter_client/multikernelmanager.py

```diff
@@ -104,7 +104,6 @@ def start_kernel(self, kernel_name=None, **kwargs):
         km = self.kernel_manager_factory(connection_file=os.path.join(
                     self.connection_dir, "kernel-%s.json" % kernel_id),
                     parent=self, log=self.log, kernel_name=kernel_name,
-                    **kwargs
```

Comment:
I haven't hunted for anything that might be relying on arbitrary kwargs passed through here, but the tests pass. Does anyone know of anything?

We're passing them again to `start_kernel` just below, so if we do rely on supplying a mixture of arguments, we should probably more clearly define which ones are for which call.